README.md

```diff
@@ -1 +1,160 @@
-# Fifa-Seminar-First-Capstone
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="./assets/FIFA_logoCroppedRemovedbg.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>FIFA Seminar (Microverse First Capstone Project)</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [ Video Description Link](#video-description)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+# 📖 FIFA Seminar (Microverse First Capstone Project) <a name="about-project"></a>
+
+**FIFA Seminar** is a website for an arbitrary event of FIFA promoting diversity and inclusion in and through football around the world.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Front-End</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+  </ul>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+  <ul>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- **This website is responsive**
+- **The website uses a number of JavaScript functions and events to increase its functonality**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Video Description <a name="video-description"></a>
+
+- [Video Description Link](https://www.loom.com/share/07cc208a7c4d4159becad32c912fec0a)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] The maximum length of the video is 5 minutes. Kindly reduce the presentation video length. :+1: